src/bundle/Resources/public/js/scripts/admin.search.filters.js

```diff
@@ -274,6 +284,58 @@
             defaultDate,
         });
     };
+    const removeSearchTag = (event) => {
+        const btn = event.target.closest(CLEAR_BTN_SELECTOR);
+        const tag = btn.closest(SEARCH_TAG_SELECTOR);
+
+        eZ.helpers.tooltips.hideAll();
+        event.target.closest('form').submit();
+        tag.remove();
+    };
+    const clearContentType = (event) => {
+        const checkbox = doc.querySelector(`#${event.target.closest(CLEAR_BTN_SELECTOR).dataset.reletedFiledId}`);
+
+        checkbox.checked = false;
+        removeSearchTag(event);
+    };
+    const clearSection = (event) => {
+        sectionSelect[0].selected = true;
+        removeSearchTag(event);
+    };
+    const clearSubtree = (event) => {
+        doc.querySelector('#search_subtree-content-breadcrumbs').hidden = true;
+        doc.querySelector('.ez-btn--udw-select-location').hidden = false;
+        subtreeInput.value = '';
+        removeSearchTag(event);
+    };
+    const clearLastModified = (event) => {
```

Comment:
`clearLastModified` and `clearLastCreated` have much in common, we can extract into a one method to be DRY